docs/api-markdown-documenter/render-api-documentation.js

```diff
@@ -55,19 +55,12 @@ async function renderApiDocumentation(inputDir, outputDir, uriRootDir, apiVersio
 		includeBreadcrumb: false, // Hugo will now be used to generate the breadcrumb
 		includeTopLevelDocumentHeading: false, // This will be added automatically by Hugo
 		createDefaultLayout: layoutContent,
-		packageFilterPolicy: (apiPackage) => {
-			// Skip `@fluid-internal` packages
+		skipPackage: (apiPackage) => {
+			// Skip `@fluid-internal` and `@fluid-private` packages
 			const packageName = apiPackage.displayName;
 			const packageScope = PackageName.getScope(packageName);
 
-			console.log(`${packageName}: ${packageScope}`);
-
-			return ["@fluid-internal"].includes(packageScope);
-		},
-		fileNamePolicy: (apiItem) => {
-			return apiItem.kind === ApiItemKind.Model
-				? "index"
-				: DefaultPolicies.defaultFileNamePolicy(apiItem);
+			return ["@fluid-internal", "@fluid-private"].includes(packageScope);
 		},
 		frontMatter: (apiItem) => createHugoFrontMatter(apiItem, config, customRenderers),
 		// TODO: enable the following once we have finished gettings the repo's release tags sorted out for 2.0.
```

Comment:
Update this comment to include `private`?

Reply:
This comment is referring to the `@internal` and `@alpha` release tags we use in comments. The change I made above was to filter out entire packages based on the scope of their name (`@fluid-private` and `@fluid-internal`), which is unrelated to release tags.